internal/verifier/verifier_test.go

```diff
@@ -182,10 +158,7 @@ func TestVerifyBuilderImageDigest_DigestFound(t *testing.T) {
 
 func TestVerifyBuilderImageDigest_DigestNotFound(t *testing.T) {
 	builderImageDigest := "9e2ba52487d945504d250de186cb4fe2e3ba023ed2921dd6ac8b97ed43e76af9"
```

Comment:
nit & pre-existing: 
```suggestion
	builderImageSHA256Digest := "9e2ba52487d945504d250de186cb4fe2e3ba023ed2921dd6ac8b97ed43e76af9"
```

or perhaps follow the Golang guideline for shorter variable names in shorter scopes, and give it a cryptic short name :D

Reply:
Done, thank you!